pkg/cmd/cmd_run.go

```diff
@@ -79,29 +79,44 @@ func NewRunCmd() *cli.Command {
 
 func KpmRun(c *cli.Context) error {
 	kclOpts := CompileOptionFromCli(c)
-	pkgWillBeCompiled := c.Args().First()
+	runEntry, errEvent := runner.FindRunEntryFrom(c.Args().Slice())
```

Comment:
Does the run command now support directly running a git's kcl package?

Reply:
This will be supported in the future. You can trace it in https://github.com/kcl-lang/kpm/issues/184.